tests/test_git_init.py

```diff
@@ -5,46 +5,28 @@
 
 import pytest
 
+from .helpers import gen_package
+
 
 @pytest.mark.parametrize(
     "git_init_cookiecutter_option",
-    ["initialise_git_repository=True", "initialise_git_repository=False"],
+    ["True", "False"],
```

Comment:
I don't like random `True`/`False`

Reply:
Agree it's not idea, but I think this is how you have to specify parmeterized tests (see [pytest docs on this](https://docs.pytest.org/en/stable/example/parametrize.html))? Happy to take suggestions for improvements though.

Reply:
You can pass in a one-element dictionary and expand it when you define the config dictionary...

```python
@pytest.mark.parametrize(
    "one_element_dictionary",
    [{"initialise_git_repo": False}, {"initialise_git_repo": True}],
)
def test_append_to_dictionary(one_element_dictionary):
    config = {
        "github_username": "test-user",
        **one_element_dictionary,
    }

```

Reply:
Obviously you don't want to call it `one_element_dictionary`...